### ISSUE TITLE
More data

### ISSUE THREAD

**Issue by vishaalagartha** [vikasagartha/agartha-backtest#11]
We need more data outside of robinhood to properly implement backtester. Consider looking into the following resources:
- [Intrinio](https://intrinio.com/financial-data/us-fundamentals-financials-metrics-ratios-stock-prices)
- [AlphaVantae](https://www.alphavantage.co/)
- [Quandl](https://blog.quandl.com/api-for-stock-data)

**Comment by vikasagartha** [comment]
relevant link:

https://www.quandl.com/api/v3/datasets/WIKI/FB/data.csv

**Comment by vishaalagartha** [comment]
Agreed, @vikasagartha . I'll create separate issues for separating the backtester and writing the data stitching. I can handle getting more data though.

**Comment by vishaalagartha** [comment]
@vikasagartha , in our current implementation, the backtester does not make any api calls actually. So all we have to do is read data from a .csv file in `main.py` and pass it into the backtester.
